src/DebuggerActions/DoesNotUnderstandDebugAction.class.st

```diff
@@ -95,7 +95,7 @@ DoesNotUnderstandDebugAction >> executeAction [
 	
 	msg := self interruptedContext tempAt: 1.
 	(msg lookupClass == UndefinedObject ) ifTrue: [ 
-		^self createMissingClassIn: self interruptedContext ].
+		self createMissingClassIn: self interruptedContext ].
```

Comment:
If we don't return here - if you have a missing class and create it won't it then continue and then try to create a method? I think I've understood that if create class fails then you assume its the nil method case - but we need to stop execution if is succeeds... so maybe something like

(self createMissingClassIn: ...) ifTrue: [ ^self ]